sanitizers/app/build.gradle

```diff
@@ -25,8 +25,16 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
+        // HWASan for devices starting from Android 14. Does no longer require a special system image.
+        // See https://developer.android.com/ndk/guides/hwasan.
         hwasan {
             initWith debug
+            debuggable true
+            packagingOptions {
+                jniLibs {
+                    useLegacyPackaging = true
```

Comment:
Is this actually needed? For wrap.sh maybe? Could you add a comment?

Reply:
Yes, sadly.